test/src/main/java/org/apache/accumulo/test/functional/ConcurrentDeleteTableIT.java

```diff
@@ -269,12 +228,53 @@ protected void doTableOp() throws Exception {
     }
   }
 
+  private abstract static class DelayedTableOp implements Runnable {
+    private final CountDownLatch cdl;
+
+    DelayedTableOp(CountDownLatch cdl) {
+      this.cdl = cdl;
+    }
+
+    @Override
+    public void run() {
+      try {
+        cdl.countDown();
+        cdl.await();
+        Thread.sleep(10);
+        doTableOp();
+      } catch (TableNotFoundException | TableOfflineException e) {
+        // expected
+      } catch (RuntimeException e) {
+        throw e;
+      } catch (Exception e) {
+        if (e.getCause().getClass().equals(ThriftTableOperationException.class)
+            && e.getMessage().equals("Table is being deleted")) {
```

Comment:
It might be useful to make `Table is being deleted` a static variable in `CompactionDriver` or `TableOperationsImpl`, then reference it from here  `CompactionDriverTest` and `TableOperationsImpl`.

Reply:
Good idea. I'll look into adding that.

Reply:
Addressed in 873e5d1. Needed to add it to `TableOperationsImpl` since `CompactionDriver` is private.